ptbr/__main__.py

```
@@ -0,0 +1,179 @@
+            raise typer.Exit(code=1) from exc
+
+        typer.echo(f"Encoding {len(labels)} labels...")
+        try:
+            embeddings = model.encode_labels(labels)
+        except NotImplementedError:
+            typer.echo(
+                f"Model '{generate_label_embeddings}' does not support label encoding "
+                f"(not a bi-encoder model). Use a bi-encoder model for label embeddings.",
+                err=True,
+            )
+            raise typer.Exit(code=1)
```

Ruff (0.15.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)